### PR TITLE
first

### PR DESCRIPTION
**Pull request by LamaDabbeet**

### CODE REVIEW

**Review by NourSammour — CHANGES_REQUESTED** [review]
Ohh, no
why did you add `hello-world` folder? you should keep the code but without that folder
instead of `hello-world/package.json` you should use `package.json`

**Comment by LamaDabbeet** [comment]
after request changes

**Comment by LamaDabbeet** [comment]
It was merged successfully before adding .gitignore , but after adding it , the automatic merge was not  applicable, should I push a version without .gitignore ??

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,5500 @@
+{
```
Comment (NourSammour) [inline package-lock.json]: please remove this file, later we will use yarn - not npm -

**Review by NourSammour — CHANGES_REQUESTED** [review]

**Comment by LamaDabbeet** [comment]
I have remove the. gitignore

File: .gitignore [hunk 2]
```diff
@@ -1 +0,0 @@
-/node_modules
```
Comment (NourSammour) [inline .gitignore:1]: Why did you remove `.gitignore`, we should keep it ?

**Review by NourSammour — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -1,11 +1,15 @@
 {
   "name": "biletiniz",
   "version": "1.0.0",
-  "description": "Uygun uçak bileti Nereye isterseniz | Biletiniz.com",
```
Comment (NourSammour) [inline package.json]: `  "description": "Uygun uçak bileti Nereye isterseniz | Biletiniz.com",`
please keep description

**Review by NourSammour — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (LamaDabbeet) [reply package.json]: done

File: .gitignore (hunk 2 above)
Comment (LamaDabbeet) [reply .gitignore:1]: done

**Review by NourSammour — APPROVED** [review]

**Pull request merged**